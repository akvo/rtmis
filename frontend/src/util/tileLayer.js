export const tileLayer = {
  url: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
};
