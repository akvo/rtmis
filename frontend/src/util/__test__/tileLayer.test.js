import { tileLayer } from "../tileLayer";

describe("util/tileLayer", () => {
  test("check the tileLayer objecr", () => {
    expect(tileLayer).toEqual(
      expect.objectContaining({
        url: "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
      })
    );
    expect(tileLayer).toMatchSnapshot();
  });
});
