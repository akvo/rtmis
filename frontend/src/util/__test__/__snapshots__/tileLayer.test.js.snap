// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`util/tileLayer check the tileLayer objecr 1`] = `
Object {
  "url": "https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png",
}
`;
